src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -849,16 +848,16 @@ public void init(boolean forceNewConnection) {
                 it.remove();
             }
 
-            ArrayList<LinkedHashMap<String, Object>> configInstances =
-                    ((ArrayList<LinkedHashMap<String, Object>>) yamlConfig.getYamlInstances());
+            ArrayList<Map<String, Object>> configInstances =
```

Comment:
I know this is outside the scope of your changes, and is more a style thing, but since we are changing to the interface with `Map` maybe we should do the same with `List`?

Reply:
This is effectively outside of the scope and it will be addressed in a next PR :)
I think there are already enough changes for a PR 😅